### ISSUE TITLE
[hirsute/impish] install hook fails - needs newer charmhelpers+ops

### ISSUE THREAD

**Issue by lourot** [openstack-charmers/charm-arista-virt-test-fixture#11]
```
     custom_handler(event)
   File "/var/lib/juju/agents/unit-arista-virt-test-fixture-0/charm/hooks/install", line 55, in on_install
     if CompareHostReleases(ubuntu_series) < 'bionic':
   File "lib/charmhelpers/core/strutils.py", line 94, in __init__
     raise KeyError("Item '{}' is not in list '{}'"
 KeyError: "Item 'hirsute' is not in list '('lucid', 'maverick', 'natty', 'oneiric', 'precise', 'quantal', 'raring', 'saucy', 'trusty', 'utopic', 'vivid', 'wily', 'xenial', 'yakkety', 'zesty', 'artful', 'bionic', 'cosmic', 'disco', 'eoan', 'focal', 'groovy')'"
```

See https://review.opendev.org/c/openstack/charm-neutron-api-plugin-arista/+/791208

**Comment by lourot** [comment]
TL;DR this probably requires some more work to update the build system to make use of `charmcraft`

Actually syncing charm-helpers leads to another issue: the `install` hook spuriously fails with 1 at the end. I suspect this has to do with the new "defered restart" functionality. One would need to also sync `ops` and `ops-openstack`. Unfortunately we're getting `ops` from an obsolete git submodule (the repo has moved). Indeed this is an early Operator charm, stemming from a time when we didn't use `charmcraft` to build them. See https://github.com/openstack-charmers/charm-arista-virt-test-fixture/blob/master/charm-init.sh#L3
